### DISCUSSION TITLE
Air raid siren has been on for days and no helicopters have spawned, is this intended?

### DISCUSSION THREAD

**Discussion by TruMonster** [TEHE-Studios/ExpandedHelicopterEvents#160] [Q&A]
Added this mod to a multiplayer server with some friends on version 41.78.16 and haven't seen any helicopters at all ever, nor the icon showing where they are. About seven days in we got a siren that hasn't gone off for about three full days. Looked for information about this mod online to see if this was intended but could find very little. If this is incorrect can someone guide me to where I can figure out how to fix it? For now I might just uninstall it since it is very annoying to have the siren on constantly. Thanks for any help.

**Comment by Chuckleberry-Finn** [comment]
This is a well-known issue. Also, issue should be posted in the issues section.

Current open issue is here: #85
